### DISCUSSION TITLE
How do I always show scrollbars?

### DISCUSSION THREAD

**Discussion by givihuda** [dbeaver/dbeaver#15810] [Q&A]
I'm having an issue with invisible appearing scrollbars obscuring things, and I'd rather just always show the scrollbars and not obscure things (as in, be in their own section, not mixed with other sections). I'm not seeing how to do this. I'm really annoyed at this trend of default invisible scrollbars.

**Comment by ShadelessFox** [comment]
Hello @givihuda,

It's not possible, unfortunately.

This behavior is OS-specific, we can't do anything about it.

**Reply by givihuda** [reply]
I've noticed in some apps that scrolling down to the end of the file keeps it separate from the hidden/showing horizontal scrollbar... is this because those apps are specifically coded to do that? Or are they "doing it right" on Mac? Or is that a happy coincidence?

I've heard elsewhere as well that it's an OS-specific limitation, but now I'm wondering because it seems half of the apps I use don't have the problem.

**Comment by Rmilkmilk** [comment]
Very late reply - This is a GTK3 setting which means you can control it in your window manager settings (KDE, GNOME, XFCE, etc.). If you don't have an option for it in your window manager, you can change it manually in your user settings file.
From the GTK3 documents here:
https://docs.gtk.org/gtk3/property.Settings.gtk-overlay-scrolling.html

~/.config/gtk-3.0/settings.ini
[Settings]
gtk-overlay-scrolling = false
